Dockerfile

```diff
@@ -25,6 +25,11 @@ RUN apt-get update && apt-get install -q -y \
       wget \
     && rm -rf /var/lib/apt/lists/*
 
+# install and run opensplice so tests will pass
+ENV RMW_IMPLEMENTATION=rmw_opensplice_cpp
+RUN apt-get update && apt-get install -q -y libopensplice69 \
+    ros-dashing-rmw-opensplice-cpp
```

Comment:
Given this package 'installed' from the [fat archive used in the nightly](https://github.com/osrf/docker_images/blob/9ee7ec18d7168ae9763c9e6f7b5cc48b5370ec87/ros2/nightly/nightly/Dockerfile#L62) build, I'm don't think blowing the nightly's version away by installing the debian (and consequently the rest of dashing from the repo) is what you'd want to do here.